proposed/extended-coding-style-guide.md

```diff
@@ -905,6 +905,68 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Compound boolean statements
+
+A compound boolean expression MAY be split across multiple lines. When splitted, 
+there MUST be only one boolean expression per line, or an other compound boolean
+expression that MUST be enclosed by parentheses. The logical operator MUST be placed 
+at the beginning of each line. 
+
+~~~php
+    return $firstBoolean
+        && ($secondBoolean || $thirdBoolean)
+        && $fourthBoolean;
+~~~
+~~~php
+    $otherBoolean = in_array($foo, [
+            'foo',
+            'bar',
+        ]) 
+        && $baz;
+~~~
+
+If the inner elements of compound expression are to be splitted, the parenthesis 
+MUST stay on the same line of the logical operator, and each inner boolean
+expression MUST be splitted with the same rules, but with an additional
+level of indentation; the closing parenthesis MUST stay on their own line.
```

Comment:
_... the closing parenthesis MUST stay on **its** own line._

Reply:
Right! Sorry, I'm not a native speaker, fixing it now!